bigquery_etl/query_scheduling/task.py

```diff
@@ -160,6 +160,8 @@ class Task:
     retry_delay: Optional[str] = attr.ib(None)
     retries: Optional[int] = attr.ib(None)
     email_on_retry: Optional[bool] = attr.ib(None)
+    sla: Optional[str] = attr.ib(None)  # todo: set to 3h
+    execution_timeout: Optional[str] = attr.ib(None)  # todo: set to 24h
```

Comment:
If the testing is successful and we decide to go for setting `sla` and `execution_timeout` for the other tasks, then we should set them to the default values in the comments